### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by xavave**
minor fixes

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -21,390 +21,397 @@
...
+						}
+					}
+
+					if (audio_choice == 0) {
+						audio = master_playlist.audio;
+						i = 0;
+						while (audio) {
+							if (0 == strcmp(audio->grp_id, selected->audio_grp)) {
+								MSG_PRINT("%d: Name: %s, Language: %s\n", i, audio->name, audio->lang ? audio->lang : "unknown");
+								i += 1;
```
Comment (xavave) [inline src/main.c:298]: i+=1 was misplaced --> put after if

File: src/main.c [hunk 2]
```diff
@@ -21,390 +21,397 @@
...
+							}
+							audio = audio->next;
+						}
+
+						MSG_PRINT("Which Language should be downloaded? ");
+						if (scanf("%d", &audio_choice) != 1 || audio_choice < 0 || audio_choice >= i) {
```
Comment (xavave) [inline src/main.c:304]: audio_choice <= 0 -->  audio_choice < 0

**Comment by selsta** [comment]
Thank you, will merge them soon

**Comment by selsta** [comment]
Thank you, merged in 2402a390415d9fde38cbaca06c38a00423b0be67

**Pull request closed without merging**